pkg/console/metrics/metrics.go

```diff
@@ -0,0 +1,88 @@
+package metrics
+
+import (
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+var (
+	consoleURL = k8smetrics.NewGaugeVec(
+		&k8smetrics.GaugeOpts{
+			Name: "console_url",
+			Help: "URL of the console exposed on the cluster",
+		},
+		[]string{"url"},
+	)
+
+	consoleBuildInfo = k8smetrics.NewGaugeVec(
+		&k8smetrics.GaugeOpts{
+			Name: "openshift_console_operator_build_info",
+			Help: "A metric with a constant '1' value labeled by major, minor, git commit & git version from which OpenShift Console Operator was built.",
+		},
+		[]string{"major", "minor", "gitCommit", "gitVersion"},
+	)
+)
+
+func init() {
```

Comment:
I believe I orinally used `init()` but then moved away from it as "best practices" indicated its better to do so (thought its been a while since I worked on this).  Timing is a potential problem that earlier examples didn't account for.

Reply:
@benjaminapetersen right but our `version.go` is still using  `init()`, also other operators (for which the metrics as build_info are working) are using this approach
Timing is not the best here I agree. So we have two options:
- try to fix it
- put on 4.5

Reply:
Fair enough, its late, this fixes the bug.  We can roll ahead, but prob should circle around and review approach again in 4.5.